bins/reth-crawler/src/main.rs

```diff
@@ -27,6 +27,10 @@ struct CrawlOpts {
     #[arg(long)]
     /// Use a sqlite db for local testing.
     local_db: bool,
+
+    /// Eth RPC url to use for getting full blocks and determining whether or not a node is synced
+    #[arg(long, default_value = "http://localhost:8545")]
+    eth_rpc_url: String,
```

Comment:
for sure - to be clear, we need to now run the crawler with`eth_rpc_url=http://69.67.151.138:8645`

Reply:
yep clarified with @Rjected in TG, lgtm